app/Modules/Newsletter/Model/Entity/Newsletter.php

```diff
@@ -0,0 +1,126 @@
+<?php declare(strict_types=1);
+
+namespace App\Modules\Newsletter\Model\Entity;
+
+use Nette\Utils\DateTime;
+
+class Newsletter
+{
+	/** @var int */
+	private $id;
+
+	/** @var string */
+	private $subject;
+
+	/** @var string */
+	private $from;
+
+	/** @var string */
+	private $introText;
+
+	/** @var string */
+	private $outroText;
+
+	/** @var string */
+	private $author;
+
+	/** @var DateTime */
+	private $created;
+
+	/**
+	 * @return int
+	 */
+	public function getId()
+	{
+		return $this->id;
+	}
+
+	/**
+	 * @return string
```

Comment:
We are still running on PHP 7, not 7.1 (rosti.cz). So if I am correct, this could throw type errors when null.